studio-docs/source/org/members.mdx

```diff
@@ -36,46 +36,76 @@ You can override a member's organization-wide role for individual graphs. For ex
 | Run schema checks                                               |     ✓     |      ✓      |      ✓      |    ✓     |          |
 | See the schemas of implementing services in federated graphs    |     ✓     |      ✓      |      ✓      |    ✓     |          |
-| View graph usage metrics                                        |     ✓     |      ✓      |      ✓      |    ✓     |          |
+| View graph usage metrics and traces                             |     ✓     |      ✓      |      ✓      |    ✓     |          |
+| Create [Development Graphs](/dev-graphs/)                       |     ✓     |      ✓      |      ✓      |    ✓     |     ✓    |
 | View graph schemas and changelogs                               |     ✓     |      ✓      |      ✓      |    ✓     |    ✓     |
 | Query graphs with the Explorer                                  |     ✓     |      ✓      |      ✓      |    ✓     |    ✓     |
 
+
+> In January 2021, we made three changes to our role structure:
+> - We removed the ability of `Observers` to view integration settings, as some of these settings contain sensitive information.
+> - We significantly reduced the permissions of the `Contributor` role, making it much closer to `Observer` plus the ability to perform a small set of write operations on protected variants.
+> - We added the `Graph Admin` role. This role is nearly identical to the former `Contributor` role; as part of the transition, we changed all existing `Contributors` to `Graph Admins`. The only difference between the old `Contributor` role and the new `Graph Admin` role is that `Graph Admins` can delete graphs (and manage graph overrides, a new feature released simultaneously).
+
+> Currently, graph usage metrics and traces are not displayed to `Consumer`s via the Studio web app but they are not blocked at the API layer. We intend to remove this capability from `Consumer`s, but for the time being you should und erstand that `Consumer`s are not fully prevented from accessing these metrics and traces.
+
 ### Billing Managers
 `Billing Managers` cannot see or manage graphs in their organizations, but they can:
 - Remove members from the organization
-- View and edit billing information
+- View and edit billing information (including changing the billing plan)
 - Manage organization configuration (name, avatar, etc.)
 
-Notable, `Billing Managers` cannot invite new members to their organizations.
+Notably, `Billing Managers` cannot invite new members to the organization or update roles.
 
 ### Dev Graphs
 
-It is worth noting that [Development Graphs](/dev-graphs/) take a special permission in our system.
-Any user, regarldess their permission, can create a dev graph and act as the `Graph Admin` for it. Dev graphs are private to the users who creat them.
+[Development Graphs](/dev-graphs/) work specially in our permissions system.
```

Comment:
I think you can say they are "a special case in our permissions system" for this to read a bit more smoothly